tools/tidy

```diff
@@ -24,9 +24,9 @@ EOF
 }
 
 set -eo pipefail
-dir="$(dirname "$0")"
+dir="$(readlink -f $(dirname "$0"))"
```

Comment:
Now you are using readlink, so calling this in os-autoinst-distri-opensuse would result in `dir=/path/to/os-autoinst-distri-opensuse/os-autoinst/tools` and
`args="--conf-file $dir/../.tidyallrc"` would then result in
`--conf-file /path/to/os-autoinst-distri-opensuse/os-autoinst/.tidyallrc` but you would want `/path/to/os-autoinst-distri-opensuse/.tidyallrc`

Reply:
In my os-autoinst-distri-opensuse fork `.perltidyrc` points to os-autoinst/.perltidyrc. But sure, I'll use realpath -s`. I suppose dependency on coreutils is ok, right?

Reply:
Updated, let's see if ready now.